src/biotite/sequence/graphics/alignment.py

```diff
@@ -445,6 +445,11 @@ def plot_alignment(axes, alignment, symbol_plotter, symbols_per_line=50,
     if seq_len % symbols_per_line != 0:
         line_count += 1
 
+    if symbol_spacing:
+        spacing_ratio = symbols_per_line / symbol_spacing
+        symbols_to_print = round(spacing_ratio) + symbols_per_line - 1
```

Comment:
I do not quite understand the aim of calculating `symbols_to_print` from `symbols_per_line`. I think a comment line would be helpful here.